app/models/ems_event.rb

```diff
@@ -196,6 +196,7 @@ def get_target(target_type)
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
     target_type = "middleware_server"   if event.event_type == "hawkular_alert"
+    target_type = "container_node"      if event.event_type == "datawarehouse_alert"
```

Comment:
This as well as the logic in container_node.evaluate_alert would be much, much easier to improve in #12773 when this is merged. Can we try to do it at that iteration please?